apps/dashboard/src/components/workflow-editor/workflow-canvas.tsx

```diff
@@ -51,40 +51,42 @@ const panOnDrag = [1, 2];
 const Y_DISTANCE = NODE_HEIGHT + 50;
 
 const mapStepToNode = (
-  step: StepResponseDto,
-  previousPosition: { x: number; y: number }
+  step: StepDto,
+  previousPosition: { x: number; y: number },
+  addStepIndex: number
 ): Node<NodeData, keyof typeof nodeTypes> => {
   let content = '';
   if (step.type === StepTypeEnum.DELAY) {
     content = `Wait to send ~ 30 minutes`;
   }
 
   return {
-    id: step.stepUuid,
+    id: crypto.randomUUID(),
```

Comment:
been available since 2015 :P did you know that?

Reply:
I think this only works in https though. Do we care?

Reply:
locally, it works as well because the local is treated as a safe context